lib/MiLight/MiLightClient.cpp

```diff
@@ -45,41 +47,45 @@ void MiLightClient::setResendCount(const unsigned int resendCount) {
   this->resendCount = resendCount;
 }
 
+
 bool MiLightClient::available() {
   if (currentRadio == NULL) {
     return false;
   }
-  
-  return currentRadio->getRadio()->available();
-}
 
+  return currentRadio->getRadioInterface()->available();
+}
 void MiLightClient::read(uint8_t packet[]) {
   if (currentRadio == NULL) {
     return;
   }
-  
-  size_t length;
-  currentRadio->getRadio()->read(packet, length);
+
+  size_t length = currentRadio->config.getPacketLength();
+
+  currentRadio->getRadioInterface()->read(packet, length);
```

Comment:
I think the interface here is supposed to be that `length` is passed as a reference, and is altered to be the length of the read packet.

We can probably just remove it. It's an artifact from Henryk's code, and I don't think it's useful anywhere.